test/test_testing.py

```diff
@@ -1,12 +1,17 @@
 def test_pytest(selenium):
-    selenium.load_package('pytest')
-    selenium.load_package('numpy')
-    selenium.load_package('nose')
-    selenium.run('from pathlib import Path')
-    selenium.run('import os')
-    selenium.run('import numpy')
-    selenium.run('base_dir = Path(numpy.__file__).parent / "core" / "tests"')
+    selenium.load_package(['pytest', 'numpy', 'nose'])
+
+    selenium.run("""
+                 from pathlib import Path
+                 import os
+                 import numpy
+                 import pytest
+
+                 base_dir = Path(numpy.__file__).parent / "core" / "tests"
+                 """)
```

Comment:
This is a real nit, but can you make this string start on the next line, indented by 4 spaces, like the examples above?

Reply:
Definitely -- fixed it.